otter/cloud_client/__init__.py

```diff
@@ -501,6 +502,15 @@ class CLBNodeLimitError(Exception):
     """
 
 
+@attr.s
+class CLBPartialNodesRemoved(Exception):
+    """
+    Exception raised when some of the nodes are removed.
+    """
+    lb_id = attr.ib()
```

Comment:
This could use an `:ivar:` documentation.  Also, perhaps a validator?  This should reference a specific type, no?  Is it unicode, or bytes?